docs/samples/v1beta1/torchserve/README.md

```diff
@@ -0,0 +1,134 @@
+# Predict on a InferenceService using Torchserve
+
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Create PV and PVC
+This document uses amazonEBS PV 
+
+### Create PV
+
+Edit volume id in pv.yaml file
+
+```
+kubectl apply -f pv.yaml
+```
+
+Expected Output
+
+```
+persistentvolume/model-pv-volume created
+```
+
+### Create PVC
+
+```
+kubectl apply -f pvc.yaml
+```
+
+Expected Output
+
+```
+persistentvolumeclaim/model-pv-claim created
+```
+
+### Create PV Pod
+
+```
+kubectl apply -f pvpod.yaml
+```
+
+Expected Output
+
+```
+pod/model-store-pod created
+```
+
+### Copy mar file and config properties to storage
+
+Download Marfile
+
+```
+wget https://torchserve.pytorch.org/mar_files/mnist.mar
```

Comment:
Can we add a few reference links about what kind of pytorch models we are able to support, like eager model and torchscript.

Reply:
@yuzisun Both eager and torchscripted models will be supported. This is part of core feature of TorchServe. So everything that works on TorchServe will work with KFServing. Developers just need to create the relevant model handlers for their models